cmd/account/show/show.go

```diff
@@ -44,6 +44,9 @@ var (
 			case npa.Account != nil:
 				// Default account is selected.
 				targetAddress = npa.Account.Address
+				if npa.AccountName != "" {
+					fmt.Printf("Name:             %s\n", npa.AccountName)
+				}
```

Comment:
Printing the name has nothing to do with the selected account. The address should be looked up in the corresponding records (wallet config, addressbook, hardcoded addresses etc.) and printed if found. Check out how we did it for `accounts transfer`.